spec/migrations/20151125081618_set_correct_sti_type_on_openstack_infra_miq_template_spec.rb

```diff
@@ -0,0 +1,83 @@
+require "spec_helper"
+require_migration
+
+describe SetCorrectStiTypeOnOpenstackInfraMiqTemplate do
+  let(:ext_management_system_stub) { migration_stub(:ExtManagementSystem) }
+  let(:miq_template_stub) { migration_stub(:Vm) }
+
+  EMS_ROW_ENTRIES = [
+    {:id => 1, :type => "ManageIQ::Providers::Openstack::InfraManager"},
+    {:id => 2, :type => "ManageIQ::Providers::Openstack::CloudManager"},
+    {:id => 3, :type => "ManageIQ::Providers::AnotherManager"}
```

Comment:
@Ladas I do not like forcing the ID for the EMS.  Can this be reworked so we save the EMSes in an array and then access them when creating the templates based on their index in the array instead of the ID?